tests/Elements/SparePartTrait.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Tests\Elements;
+
+use \Shoprunback\Elements\SparePart;
+
+trait SparePartTrait
+{
+    public static function getElementClass()
+    {
+        return 'Shoprunback\Elements\SparePart';
+    }
+
+    public static function createDefault()
+    {
+        $name = self::randomString();
+        $reference = self::randomString();
```

Comment:
These 2 variables are not necessary. 
`$sparePart->name = self::randomString();`

Reply:
Its for unittest to generate value for varaible.